assets/css/components/_notifications.scss

```diff
@@ -97,3 +97,14 @@
     }
   }
 }
+
+//first-login-modal-eth
+.modal-dialog {
+  .modal-content {
+    .modal-body {
+      > img {
+        height: 150px;
```

Comment:
This will change all the first children of modal-body's throughout the site to 150px. Is that intended?